inference-engine/src/low_precision_transformations/src/fake_quantize.cpp

```diff
@@ -91,11 +90,11 @@ bool FakeQuantizeTransformation::checkElementwise(const std::shared_ptr<Node>& e
 
     Shape shape = constant->get_output_shape(0);
     if ((!shape.empty()) && (shape_size(shape) != 1ul)) {
-        if ((eltwiseOutputShape.size() - shape.size()) > 1) {
+        if ((eltwiseOutputPShape.rank().get_length() - shape.size()) > 1) {
```

Comment:
are we sure?

Reply:
Yes. Check on dynamic rank is [above](https://github.com/openvinotoolkit/openvino/pull/6129/files#diff-0ed65eaabe9b6f05a36ecc74912e2d72d9a02c329d6bbdb84fe4a8ce076e47a4R82)